WcaOnRails/spec/views/registrations/psych_sheet_event.html.erb_spec.rb

```diff
@@ -25,22 +27,41 @@
       countryRank: 1,
     )
 
+    registrations << FactoryGirl.create(:registration, :accepted, user: user2, competition: competition)
+    RanksAverage.create!(
+      personId: registrations.last.personId,
+      eventId: "333",
+      best: "4242",
+      worldRank: 10,
+      continentRank: 10,
+      countryRank: 10,
+    )
+    RanksSingle.create!(
+      personId: registrations.last.personId,
+      eventId: "333",
+      best: "3334",
+      worldRank: 23,
+      continentRank: 23,
+      countryRank: 23,
+    )
+
     # Someone who has never competed before.
-    registrations << FactoryGirl.create(:registration, user: FactoryGirl.create(:user), competition: competition)
+    registrations << FactoryGirl.create(:registration, :accepted, user: FactoryGirl.create(:user), competition: competition)
 
     # Someone who has not competed in 333 before.
-    registrations << FactoryGirl.create(:registration, competition: competition)
-    registrations.last.tied_previous = true
+    registrations << FactoryGirl.create(:registration, :accepted, competition: competition)
```

Comment:
Hmm I'm a bit confused by the comment above. The registration has user with wca_id (created in the factory), thus it has a person, but this persons hasn't any results. So actually it's someone who has never competed before =) (as above). Maybe we should provide better situation / comment. What do you think?